### ISSUE TITLE
Failing testsuite gce-device-plugin-gpu-1.11

### ISSUE THREAD

**Issue by foxish** [kubernetes/kubernetes#68372]
This testsuite is failing or very flaky and blocking 1.11.3. 
Can someone triage the issue?

@yguo0905 @mindprince 
https://k8s-testgrid.appspot.com/sig-release-1.11-blocking#gce-device-plugin-gpu-1.11

**Comment by foxish** [comment]
@kubernetes/sig-node-bugs

**Comment by mindprince** [comment]
cc @jiayingz 

The failure doesn't look related to this test. Are the other tests passing?

**Comment by mindprince** [comment]
https://k8s-testgrid.appspot.com/wg-resource-management#Summary
The GPU test is passing for 1.10, 1.12 and master. As far as I know nothing GPU related went into 1.11 (but I haven't been keeping up). @jiayingz will know for sure.

**Comment by guineveresaenger** [comment]
This failure is blocking a fix for master upgrade tests that are important for this release. If I followed the rabbit hole correctly. Further updates would be much appreciated. :)

**Comment by jiayingz** [comment]
I don't think this test failure would prevent merging any PRs.

**Comment by BenTheElder** [comment]
> I don't think this test failure would prevent merging any PRs.

Possibly not, but release test suites failing block releases from being published, and upgrade tests depend on the previous release builds IIRC...

**Comment by yguo0905** [comment]
Looks like the same problem is blocking https://github.com/kubernetes/kubernetes/pull/68381.

**Comment by yguo0905** [comment]
From https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-device-plugin-gpu-stable1/2425/artifacts/bootstrap-e2e-master/serial-1.log, master was restarted before the error was printed.

```
[  301.771969] cbr0: port 2(veth8ccf2657) entered forwarding state
[  301.778047] cbr0: port 2(veth8ccf2657) entered forwarding state
[  316.807804] cbr0: port 2(veth8ccf2657) entered forwarding state
[  537.313961] sysrq: SysRq : Resetting
SeaBIOS (version 1.8.2-20180718_090847-google)
Total RAM Size = 0x00000000f0000000 = 3840 MiB
CPUs found: 1     Max CPUs supported: 256
found virtio-scsi at 0:3
virtio-scsi vendor='Google' product='PersistentDisk' rev='1' type=0 removable=0
virtio-scsi blksize=512 sectors=41943040 = 20480 MiB
virtio-scsi vendor='Google' product='PersistentDisk' rev='1' type=0 removable=0
virtio-scsi blksize=512 sectors=41943040 = 20480 MiB
drive 0x000f2280: PCHS=0/0/0 translation=lba LCHS=1024/255/63 s=41943040
drive 0x000f2240: PCHS=0/0/0 translation=lba LCHS=1024/255/63 s=41943040
Booting from Hard Disk 0...
```

**Comment by jiayingz** [comment]
As Yang pointed out, somehow the master node was rebooted, which can also be seen from https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-device-plugin-gpu-stable1/2404/artifacts/bootstrap-e2e-master/kube-master-configuration.log:
Sep 01 01:36:05.090088 bootstrap-e2e-master systemd[1]: Started Configure kubernetes master.
-- Reboot --
Sep 01 01:44:19.642160 bootstrap-e2e-master systemd[1]: Starting Configure kubernetes master...

I didn't see any problem in other master logs so far. It is strange that we are only seeing the problem in this gpu test because the master node setting of this test should be the same as other regular 1.11 gce tests. The gpu test does have special node configuration settings as specified in jobs/env/ci-kubernetes-e2e-gce-device-plugin-gpu.env. Looks like we are still using KUBE_GCE_NODE_IMAGE=gke-1-8-4-gke-1-cos-stable-63-10032-71-0-p-v171208-pre-nvda-gpu cos version for GPU nodes. @mindprince do you think if we should use a newer cos prebuilt version? However, this gpu node setting shouldn't affect master node and I am still puzzled why the master node of this test was restarted.

Btw, this test failure shouldn't block any PR merging. For people who are wondering whether their PRs are blocked by this test failure, please check whether your PRs have do-not-merge/cherry-pick-not-approved label, which is likely the actual reason blocking your PR.

**Comment by mindprince** [comment]
/unassign
(To reflect that I am not actively working on this)

/assign @jiayingz

**Comment by jiayingz** [comment]
FYI, the failed ci-kubernetes-e2e-gce-device-plugin-gpu-stable1 test is running in the k8s-jkns-e2e-gke-gpus-beta project. @krzyzacy and @BenTheElder is it possible that we are trying to run too many gpu tests in this project at the same time that somehow caused the underlying testing infrastructure to turn off existing testing clusters in this project to start new ones?

**Comment by krzyzacy** [comment]
cc @Random-Liu @yujuhong 

compared the docker log between 1.11 and 1.12, I found on 1.11 there's a lot of 

```
time="2018-09-06T20:06:53.287436660Z" level=info msg="libcontainerd: new containerd process, pid: 662"
time="2018-09-06T20:06:55.785892169Z" level=info msg="Graph migration to content-addressability took 0.00 seconds"
time="2018-09-06T20:06:55.788947914Z" level=info msg="Loading containers: start."
time="2018-09-06T20:06:56.512300045Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (4ab86da9b769e2339f881cc608718c5ae7f59bdf3e9792460d3ebae9ae419e68)."
time="2018-09-06T20:06:56.518400972Z" level=warning msg="libcontainerd: failed to retrieve container 4ab86da9b769e2339f881cc608718c5ae7f59bdf3e9792460d3ebae9ae419e68 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:56.518857097Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/4ab86da9b769e2339f881cc608718c5ae7f59bdf3e9792460d3ebae9ae419e68/shm: invalid argument"
time="2018-09-06T20:06:56.544775498Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (32c0fa02684d2f1d6c7357cc3d6c905e94d7d4c713ad32592399c224260b1a4e)."
time="2018-09-06T20:06:56.545624221Z" level=warning msg="libcontainerd: failed to retrieve container 32c0fa02684d2f1d6c7357cc3d6c905e94d7d4c713ad32592399c224260b1a4e state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:56.546081796Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/32c0fa02684d2f1d6c7357cc3d6c905e94d7d4c713ad32592399c224260b1a4e/shm: invalid argument"
time="2018-09-06T20:06:56.564962752Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (37f9a952bc7c7672e28d42096272790dc347fb9fe205ec6ac9e2226a0fb0a30b)."
time="2018-09-06T20:06:56.565989996Z" level=warning msg="libcontainerd: failed to retrieve container 37f9a952bc7c7672e28d42096272790dc347fb9fe205ec6ac9e2226a0fb0a30b state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:56.567374043Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/37f9a952bc7c7672e28d42096272790dc347fb9fe205ec6ac9e2226a0fb0a30b/shm: invalid argument"
time="2018-09-06T20:06:56.587999116Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (b44ce2feb2094c220cd0ec7971095601c6b8f4aadeacbe6b0773db1d4048df7e)."
time="2018-09-06T20:06:56.588844598Z" level=warning msg="libcontainerd: failed to retrieve container b44ce2feb2094c220cd0ec7971095601c6b8f4aadeacbe6b0773db1d4048df7e state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:56.589351496Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/b44ce2feb2094c220cd0ec7971095601c6b8f4aadeacbe6b0773db1d4048df7e/shm: invalid argument"
time="2018-09-06T20:06:56.627118236Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (4df85b9f6640e062329cd48c1d75514756f0b334b635412308b365c4ae47b731)."
time="2018-09-06T20:06:56.627855212Z" level=warning msg="libcontainerd: failed to retrieve container 4df85b9f6640e062329cd48c1d75514756f0b334b635412308b365c4ae47b731 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:56.648485984Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (2aa8e2bbae1ff41b00996844a7f2a77a96f3a91f963b17211fe6727f9c908363)."
time="2018-09-06T20:06:56.649241188Z" level=warning msg="libcontainerd: failed to retrieve container 2aa8e2bbae1ff41b00996844a7f2a77a96f3a91f963b17211fe6727f9c908363 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:56.649716515Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/2aa8e2bbae1ff41b00996844a7f2a77a96f3a91f963b17211fe6727f9c908363/shm: invalid argument"
time="2018-09-06T20:06:56.658144355Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (d3c2fe1c39182f5f940c739e34f20251e66435f3cd68fb7b8dfb39ad16a8b21b)."
time="2018-09-06T20:06:56.659314776Z" level=warning msg="libcontainerd: failed to retrieve container d3c2fe1c39182f5f940c739e34f20251e66435f3cd68fb7b8dfb39ad16a8b21b state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:56.659774716Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/d3c2fe1c39182f5f940c739e34f20251e66435f3cd68fb7b8dfb39ad16a8b21b/shm: invalid argument"
time="2018-09-06T20:06:56.683078802Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (259a441bf67e080213a789243809db7ff6652687c9c75b4a23fd82c152519120)."
time="2018-09-06T20:06:56.686910162Z" level=warning msg="libcontainerd: failed to retrieve container 259a441bf67e080213a789243809db7ff6652687c9c75b4a23fd82c152519120 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:56.688515911Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/259a441bf67e080213a789243809db7ff6652687c9c75b4a23fd82c152519120/shm: invalid argument"
time="2018-09-06T20:06:57.003427993Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (d8d067de669ddc11083690ef42d28b3838d31e04b766c6098b3a67ed0188b872)."
time="2018-09-06T20:06:57.005446503Z" level=warning msg="libcontainerd: failed to retrieve container d8d067de669ddc11083690ef42d28b3838d31e04b766c6098b3a67ed0188b872 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.029737913Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (fef7e4337467bd19be7a7ff68f1fb13a8989ca2cf72390607595c8dae06ec407)."
time="2018-09-06T20:06:57.030692702Z" level=warning msg="libcontainerd: failed to retrieve container fef7e4337467bd19be7a7ff68f1fb13a8989ca2cf72390607595c8dae06ec407 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.038505404Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (677f8b976f213b4d0aa5da1945d2cd2e781e10ca17c6fa05285b9a32fc849a61)."
time="2018-09-06T20:06:57.040148390Z" level=warning msg="libcontainerd: failed to retrieve container 677f8b976f213b4d0aa5da1945d2cd2e781e10ca17c6fa05285b9a32fc849a61 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.049191645Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (976f82926a07611673519fb8eac9637be87f41cb8d28ae75f354c71fe15e2252)."
time="2018-09-06T20:06:57.070233268Z" level=warning msg="libcontainerd: failed to retrieve container 976f82926a07611673519fb8eac9637be87f41cb8d28ae75f354c71fe15e2252 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.084160261Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (a950c25ebcd733283e8fa02fb67caa7c83e0bf5661d5587da601b1f74b4a6f52)."
time="2018-09-06T20:06:57.085046284Z" level=warning msg="libcontainerd: failed to retrieve container a950c25ebcd733283e8fa02fb67caa7c83e0bf5661d5587da601b1f74b4a6f52 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.147186129Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (864adac6cda52e18a2a9f78ec11de342ca09843c191acad0f602615d06623149)."
time="2018-09-06T20:06:57.150176954Z" level=warning msg="libcontainerd: failed to retrieve container 864adac6cda52e18a2a9f78ec11de342ca09843c191acad0f602615d06623149 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.160421328Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (8a99b3bc27f342156c4e49813f9feb56d08dc34d2adfba7d4f51a17e4e737b74)."
time="2018-09-06T20:06:57.165106169Z" level=warning msg="libcontainerd: failed to retrieve container 8a99b3bc27f342156c4e49813f9feb56d08dc34d2adfba7d4f51a17e4e737b74 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.165669345Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/8a99b3bc27f342156c4e49813f9feb56d08dc34d2adfba7d4f51a17e4e737b74/shm: invalid argument"
time="2018-09-06T20:06:57.203836247Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (81cf7376dfc347129005eb2163b5873cd875a709591e5045fb2a3b31887a0828)."
time="2018-09-06T20:06:57.204688863Z" level=warning msg="libcontainerd: failed to retrieve container 81cf7376dfc347129005eb2163b5873cd875a709591e5045fb2a3b31887a0828 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.255421843Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (60c5fc840a0bc80b04974d165638ce0e311e0355ee5f750bc69c6f1550891f38)."
time="2018-09-06T20:06:57.258720863Z" level=warning msg="libcontainerd: failed to retrieve container 60c5fc840a0bc80b04974d165638ce0e311e0355ee5f750bc69c6f1550891f38 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.284754584Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (f659b9cbf8eec0474037c111fed53c332c8db8d05d4905775707aa92ee63f068)."
time="2018-09-06T20:06:57.285603528Z" level=warning msg="libcontainerd: failed to retrieve container f659b9cbf8eec0474037c111fed53c332c8db8d05d4905775707aa92ee63f068 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.285996932Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/f659b9cbf8eec0474037c111fed53c332c8db8d05d4905775707aa92ee63f068/shm: invalid argument"
time="2018-09-06T20:06:57.300938999Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (9feed940d3fdf624cefcb2607ec1221ebebc862ff44ec2344d434a479688697f)."
time="2018-09-06T20:06:57.305173232Z" level=warning msg="libcontainerd: failed to retrieve container 9feed940d3fdf624cefcb2607ec1221ebebc862ff44ec2344d434a479688697f state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.305728581Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/9feed940d3fdf624cefcb2607ec1221ebebc862ff44ec2344d434a479688697f/shm: invalid argument"
time="2018-09-06T20:06:57.358675280Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (8acd98c4ab2a5603a7d3d2929b0b510b91e9ee8ce04d5604d544cf8886277443)."
time="2018-09-06T20:06:57.359821396Z" level=warning msg="libcontainerd: failed to retrieve container 8acd98c4ab2a5603a7d3d2929b0b510b91e9ee8ce04d5604d544cf8886277443 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.399148631Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (3239f52cc393831e3269d4d4faccb41e4bf02b76cbdd4c4937b1adb96080a56c)."
time="2018-09-06T20:06:57.399499003Z" level=warning msg="libcontainerd: failed to retrieve container 3239f52cc393831e3269d4d4faccb41e4bf02b76cbdd4c4937b1adb96080a56c state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.408682866Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (f1523648b1a0a00f1ffd68d7f134a02d6f153e2ccc9d2e1a66c2cb2ebd512a24)."
time="2018-09-06T20:06:57.410494421Z" level=warning msg="libcontainerd: failed to retrieve container f1523648b1a0a00f1ffd68d7f134a02d6f153e2ccc9d2e1a66c2cb2ebd512a24 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.420336009Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (a323413b6132974722841daecc92b84fb61a0132442a166424f40fd152213e3e)."
time="2018-09-06T20:06:57.426097139Z" level=warning msg="libcontainerd: failed to retrieve container a323413b6132974722841daecc92b84fb61a0132442a166424f40fd152213e3e state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.426667429Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/a323413b6132974722841daecc92b84fb61a0132442a166424f40fd152213e3e/shm: invalid argument"
time="2018-09-06T20:06:57.449382509Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (24caae793649320c6f43b1d165f08141c386f8776ba9f1436eeb26f6049e336f)."
time="2018-09-06T20:06:57.451065629Z" level=warning msg="libcontainerd: failed to retrieve container 24caae793649320c6f43b1d165f08141c386f8776ba9f1436eeb26f6049e336f state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.451575297Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/24caae793649320c6f43b1d165f08141c386f8776ba9f1436eeb26f6049e336f/shm: invalid argument"
time="2018-09-06T20:06:57.481625975Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (6d42e3c4c7b0825dfb31f2088f54e749a8214ebf201a0d0aa87b7e4cfd786ab6)."
time="2018-09-06T20:06:57.482416270Z" level=warning msg="libcontainerd: failed to retrieve container 6d42e3c4c7b0825dfb31f2088f54e749a8214ebf201a0d0aa87b7e4cfd786ab6 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.517186153Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (40a4560733e4ba6b28246f65ee4c545dc34e084f6e8efb066d8fe927a69c9a43)."
time="2018-09-06T20:06:57.518083789Z" level=warning msg="libcontainerd: failed to retrieve container 40a4560733e4ba6b28246f65ee4c545dc34e084f6e8efb066d8fe927a69c9a43 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.529709609Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/40a4560733e4ba6b28246f65ee4c545dc34e084f6e8efb066d8fe927a69c9a43/shm: invalid argument"
time="2018-09-06T20:06:57.549763179Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (d372f1ee0f1d027b58b918731bdfba47ae5dc0706f9c855fab5864b8d5e15654)."
time="2018-09-06T20:06:57.552481857Z" level=warning msg="libcontainerd: failed to retrieve container d372f1ee0f1d027b58b918731bdfba47ae5dc0706f9c855fab5864b8d5e15654 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.586779019Z" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (60af66eb439f18133e715b404ba9090f40f7205133881f5396c0d0777aa53c15)."
time="2018-09-06T20:06:57.587631603Z" level=warning msg="libcontainerd: failed to retrieve container 60af66eb439f18133e715b404ba9090f40f7205133881f5396c0d0777aa53c15 state: rpc error: code = 2 desc = containerd: container not found"
time="2018-09-06T20:06:57.659339974Z" level=info msg="Firewalld running: false"
time="2018-09-06T20:06:57.997106671Z" level=info msg="Removing stale sandbox 7d252426d4ecb13b9238e40b8b900de51c8a900847f483f0b08e56e7d8d57f33 (b44ce2feb2094c220cd0ec7971095601c6b8f4aadeacbe6b0773db1d4048df7e)"
time="2018-09-06T20:06:58.078129618Z" level=info msg="Removing stale sandbox e992d016314ed7f2eeeeccc57305e6773a3d6576e6dda1eb87a711e6868aea64 (37f9a952bc7c7672e28d42096272790dc347fb9fe205ec6ac9e2226a0fb0a30b)"
time="2018-09-06T20:06:58.139119798Z" level=info msg="Removing stale sandbox 342221155f38b8c0fca0c7ce46a5737eccaedca810cc52a2a21d0eadeb7d7c36 (9feed940d3fdf624cefcb2607ec1221ebebc862ff44ec2344d434a479688697f)"
time="2018-09-06T20:06:58.201079912Z" level=info msg="Removing stale sandbox 5e857586b4867373715855560f35c0fd7465b83fc5903cac35974da0005842ba (4ab86da9b769e2339f881cc608718c5ae7f59bdf3e9792460d3ebae9ae419e68)"
time="2018-09-06T20:06:58.261101860Z" level=info msg="Removing stale sandbox 28db6fa2f0d67ac0e063f5508b9cb2003e9b696cc69bae603e058ee2e558f89c (40a4560733e4ba6b28246f65ee4c545dc34e084f6e8efb066d8fe927a69c9a43)"
time="2018-09-06T20:06:58.321120852Z" level=info msg="Removing stale sandbox 56dc49da68c0e63f3aa8298ce985adbf4678973dfa11f82d595b52131cc22c1a (24caae793649320c6f43b1d165f08141c386f8776ba9f1436eeb26f6049e336f)"
time="2018-09-06T20:06:58.385528048Z" level=info msg="Removing stale sandbox 83b0761fdf0fa5787ee481642b4f58af4fda94752e2049889f93a059be4a7fc4 (2aa8e2bbae1ff41b00996844a7f2a77a96f3a91f963b17211fe6727f9c908363)"
time="2018-09-06T20:06:58.465019754Z" level=info msg="Removing stale sandbox 97dad60bd78bc06c5a946609ca88c5b0ed6a7f36a77f7c3e4fd0957b3765e87f (32c0fa02684d2f1d6c7357cc3d6c905e94d7d4c713ad32592399c224260b1a4e)"
time="2018-09-06T20:06:58.526080139Z" level=info msg="Removing stale sandbox a96dc8f73c7bde4f6d935ac65c73ad4b6b0847a94dca5c8a84a59e33efdf0d71 (8a99b3bc27f342156c4e49813f9feb56d08dc34d2adfba7d4f51a17e4e737b74)"
time="2018-09-06T20:06:58.583138602Z" level=info msg="Removing stale sandbox dfb1d305d7abae1352e0d380b77eb371f20da5d1966a26eb2b25f5baccb0a5a7 (d3c2fe1c39182f5f940c739e34f20251e66435f3cd68fb7b8dfb39ad16a8b21b)"
time="2018-09-06T20:06:58.643093575Z" level=info msg="Removing stale sandbox 0b1319a4a1962683fea83f1d3718e6fecd83c5888809b56babe229bf375440a5 (259a441bf67e080213a789243809db7ff6652687c9c75b4a23fd82c152519120)"
time="2018-09-06T20:06:58.702327106Z" level=info msg="Removing stale sandbox 1d3cd1d00ec9150477e8aab11f7283344d4fca081eb218618d8227253083bf5a (a323413b6132974722841daecc92b84fb61a0132442a166424f40fd152213e3e)"
time="2018-09-06T20:06:58.761096895Z" level=info msg="Removing stale sandbox f0baf839d2f2b825a445ed3ab710c61cae7003cfe2ac07e5618ecb94dd263eb9 (f659b9cbf8eec0474037c111fed53c332c8db8d05d4905775707aa92ee63f068)"
time="2018-09-06T20:06:58.803454676Z" level=info msg="Default bridge (docker0) is assigned with an IP address 169.254.123.0/24. Daemon option --bip can be used to set a preferred IP address"
time="2018-09-06T20:06:58.964233924Z" level=info msg="Loading containers: done."
time="2018-09-06T20:06:59.276482764Z" level=info msg="Daemon has completed initialization"
time="2018-09-06T20:06:59.277024719Z" level=info msg="Docker daemon" commit=f5ec1e2 graphdriver=overlay2 version=17.03.2-ce
time="2018-09-06T20:06:59.314008497Z" level=info msg="API listen on /var/run/docker.sock"
Started Docker Application Container Engine.
Stopping Docker Application Container Engine...
time="2018-09-06T20:07:54.752274645Z" level=info msg="Processing signal 'terminated'"
time="2018-09-06T20:07:54.757834152Z" level=info msg="stopping containerd after receiving terminated"
docker daemon exited
Stopped Docker Application Container Engine.
Starting Docker Application Container Engine...
```

any clue if containerd has some new changes to 1.11? :-)

**Comment by BenTheElder** [comment]
> is it possible that we are trying to run too many gpu tests in this project at the same time that somehow caused the underlying testing infrastructure to turn off existing testing clusters in this project to start new ones?

Discussed offline: but to be clear here, the projects are "leased" from a manager (boskos in test-infra) for each run, the testing has exclusive access to that GCP project.

**Comment by krzyzacy** [comment]
> Discussed offline: but to be clear here, the projects are "leased" from a manager (boskos in test-infra) for each run, the testing has exclusive access to that GCP project.

also if that's the case, it should affect *all* the gpu tests, rather than just 1.11 branch

**Comment by foxish** [comment]
Please update this issue if there are any new updates on resolving these failures.

**Comment by jiayingz** [comment]
@yguo0905 pointed me to this PR 
https://github.com/kubernetes/kubernetes/pull/67590 that was merged into 1.11 around the time when the test started to fail. I reproduced the test failure in 1.11 release from local runs and verified that with https://github.com/kubernetes/kubernetes/pull/67590 reverted, the test passed. So looks like this PR reducing API server cpu resource has caused master node reboot, though I am not sure why. Couldn't find any system logs that could provide a hint on why the master node reboot. What I saw is master node just rebooted, and it is not an instance recreate as the master node kept the same instance_id.

**Comment by krzyzacy** [comment]
/shrug

https://github.com/kubernetes/kubernetes/pull/68229

which has not merged...

**Comment by foxish** [comment]
Fixed by the revert

**Comment by jiayingz** [comment]
I don't think the driver installer was actually doing anything. On master node, the driver installer should just sleep after detecting there is no gpu attached.

**Comment by vishh** [comment]
I don't think we should even be running the driver installer on the master unless a GPU is attached to the master.

**Comment by vishh** [comment]
@krzyzacy @jiayingz opened https://github.com/kubernetes/kubernetes/pull/68492 to prevent driver installer from running on the master in the first place.
